string.go

```diff
@@ -57,17 +60,16 @@ func AESDecrypt(key, nonce, ciphertext []byte) ([]byte, error) {
 		return nil, err
 	}
 
-	plainText, err := gcm.Open(nil, nonce, ciphertext, nil)
-	if err != nil {
-		return nil, err
-	}
+	size := gcm.NonceSize()
+	nonce := ciphertext[:size]
```

Comment:
Well, when you're decrypting a text using `AESGCMDecrypt` you don't know whether this text was provided by an attacker which could have just truncated it to one byte. The app would crash then.